Tools/AP_Periph/Parameters.cpp

```diff
@@ -298,23 +298,43 @@ const AP_Param::Info AP_Periph_FW::var_info[] = {
 #endif
 
 #ifdef HAL_PERIPH_ENABLE_RANGEFINDER
-    // @Param: RNGFND_BAUDRATE
+    // @Param: RNGFND1_BAUDRATE
```

Comment:
i'd prefer (with some regret) keeping this as RNGFND_BAUDRATE

Reply:
Done